Makefile

```diff
@@ -224,29 +217,15 @@ $(ENVTEST): $(LOCALBIN)
 .PHONY: protoc
 protoc: $(PROTOC) ## download and install protoc.
 $(PROTOC): $(LOCALBIN)
-	@ test -s $(LOCALBIN)/protoc || \
-	( curl -L https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-$(PROTOC_PLATFORM).zip \
-		-o /tmp/protoc-${PROTOC_VERSION}-$(PROTOC_PLATFORM).zip && \
-	unzip /tmp/protoc-${PROTOC_VERSION}-$(PROTOC_PLATFORM).zip -d /tmp/protoc/ && \
-	cp /tmp/protoc/bin/protoc $(LOCALBIN)/protoc && \
-	rm -rf /tmp/protoc/ && \
-	rm /tmp/protoc-${PROTOC_VERSION}-$(PROTOC_PLATFORM).zip && \
-	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@${PROTOC_GEN_GO_GRPC_VERSION} && \
-	go install google.golang.org/protobuf/cmd/protoc-gen-go@$(PROTOC_GEN_GO_VERSION) )
-
-.PHONY: skaffold
-skaffold:
-	@ test -s $(LOCALBIN)/skaffold || \
-	( curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-$(SKAFFOLD_PLATFORM) && \
-	chmod +x skaffold && \
-	mv skaffold $(LOCALBIN)/skaffold )
-
-.PHONY: envsubst
-envsubst:
-	@ test -s $(LOCALBIN)/envsubst || \
-	( curl -L https://github.com/a8m/envsubst/releases/download/v1.2.0/envsubst-`uname -s`-`uname -m` -o envsubst && \
-	chmod +x envsubst && \
-	mv envsubst $(LOCALBIN)/envsubst )
```

Comment:
Looks like you ripped out skaffold and envsubs in the process. On purpose?

Reply:
Whoops, accident, I made this change in another branch and brought it into main in a sketchy way. Will fix.